host_ip.txt

```diff
@@ -0,0 +1,6 @@
+192.168.1.108
+192.168.0.102
+192.168.0.103
+192.168.0.104
+192.168.0.100
+192.168.0.106
```

Comment:
it does not make sense to push this file to github since it will always be programmatically recreated.
I would suggest adding `host_ip.txt` to the gitignore.
Please also ensure that `deploy_code_to_rpi.sh` also deletes this file before it tries to recreate it!

Reply:
Added host_ip.txt in .gitignore.